### PR TITLE
operators [CI] t8c (8.2.0)

### PR DESCRIPTION
**Pull request by esara**
Signed-off-by: esara <endre.sara@turbonomic.com>

### CODE REVIEW

File: operators/t8c/8.2.0/xls.charts.helm.k8s.io.crd.yaml [hunk 1]
```diff
@@ -0,0 +1,3905 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (camilamacedo86) [inline operators/t8c/8.2.0/xls.charts.helm.k8s.io.crd.yaml:1]: Hi just would like to let you know that this version will not work for OCP 4.9 or k8s 1.22. 
See: https://github.com/turbonomic/t8c-install/issues/12

Comment (esara) [reply operators/t8c/8.2.0/xls.charts.helm.k8s.io.crd.yaml:1]: hi @camilamacedo86 , yes I know

**Comment by J0zi** [comment]
@esara the pipeline was unable to pull your operator image. Restarting, in case of accidental. If fails again, please check.

**Comment by esara** [comment]
@J0zi yes, I saw the same - we are just unlucky, it is the same image that is being pulled for the 4.6 and 4.7 certification, I think we are just randomly being throttled by docker, when the 4.8 certification runs - i will try again

**Comment by esara** [comment]
this operator is published in dockerhub (as turbonomic/t8c-operator:8.2)
https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/181/files#diff-44ce54c6faef6932974b0ceb98729eff937eb2ca5431a4f400c93978f3cbb6feR59

why is the openshift deploy test trying to pull the images from the redhat registry?
`     Events:
      Type     Reason          Age                 From               Message
      ----     ------          ----                ----               -------
      Normal   Scheduled       16m                 default-scheduler  Successfully assigned test-operators/t8c-operator-7cb7f84dc-n2fcq to ip-10-0-190-124.ec2.internal
      Normal   AddedInterface  16m                 multus             Add eth0 [10.129.2.18/23] from openshift-sdn
      Normal   Pulling         14m (x4 over 16m)   kubelet            Pulling image "turbonomic/t8c-operator:8.2"
      Warning  Failed          14m (x4 over 16m)   kubelet            Failed to pull image "turbonomic/t8c-operator:8.2": rpc error: code = Unknown desc = Error reading manifest 8.2 in registry.redhat.io/turbonomic/t8c-operator: unknown: Not Found
      Warning  Failed          14m (x4 over 16m)   kubelet            Error: ErrImagePull
      Warning  Failed          14m (x6 over 16m)   kubelet            Error: ImagePullBackOff
      Normal   BackOff         78s (x64 over 16m)  kubelet            Back-off pulling image "turbonomic/t8c-operator:8.2"
`
I can publish this in the partner registry, but as an ISV I dont I can publish this in the redhat registry...

**Review by boconnell2210 — APPROVED** [review]

**Pull request merged**